portfolio/src/main/webapp/style.css

```diff
@@ -115,4 +116,9 @@
   position: absolute;
   top: 10px;
   right: 10px;
+}
+
+.row {
+  margin-left: 0 !important;
+  margin-right: 0 !important;
```

Comment:
Try to avoid using important whenever possible. If you're trying to override style that is being applied to descendant elements, you can use css descendant selector. For example, use `.row. children-class` to make it higher precedence than styles of `.children-class`.

Reply:
Sounds good, thanks.